lib/departure/option.rb

```diff
@@ -8,10 +8,7 @@ class Option
     # @param string [String]
     # @return [Option]
     def self.from_string(string)
-      pair = string.split('=')
-      name = pair[0][2..-1]
-      value = pair[1]
-
+      name, value = string.split(/\s|=/, 2)
```

Comment:
> We basically strip the formatting out so that we can either read from CLI or pass a key and a value by hand to the constructor like in https://github.com/redbooth/departure/blob/master/lib/percona_migrator/cli_generator.rb#L17

`Option::new` should work as before. The main change with `Option` is that `#name` now is the "normalized option" name:

* `Option.new('arg').name == '--arg' # this was 'arg' before this change`
* `Option.new('--arg').name == '--arg' # same as before`
* `Optoon.new('-a').name == '-a' # same as before`

Is there any reason we want to keep `Option#name` in the `arg` (and not `--arg`) format?

> This means we would only need to call #normalize_option when executing #to_s.

`#name` is used by `#==` and `#hash`, so I thought to cache the normalized value on initialization. Otherwise, we'll have to call `#normalize_option` in those methods, too.